demo/src/lib.rs

```diff
@@ -344,6 +342,23 @@ pub fn run(args: &DemoArgs) -> RafxResult<()> {
             }
         }
 
+        //
+        // Process input
+        //
+        if !process_input(&mut scene_manager, &mut world, &resources, &mut event_pump) {
+            break 'running;
+        }
+
+        {
+            let mut viewports_resource = resources.get_mut::<ViewportsResource>().unwrap();
+            let (width, height) = sdl2_systems.window.vulkan_drawable_size();
```

Comment:
@aclysma Should this be `size()` instead of `vulkan_drawable_size()`?

Reply:
I'm genuinely not sure. I made this change because the imgui code was asking for a viewport size bigger than the window. I thought there was a mismatch between size() and vulkan_drawable_size(). However, the root cause was that I was ticking imgui before sdl event processing, and then doing rendering. The size changed when processing sdl events. So imgui's window size and the swapchain size did not match.

I think the difference between the two is how hidpi monitors are handled. To some degree, that might be an opinion more than a "right/wrong" thing.